src/components/button/_button.scss

```diff
@@ -80,14 +81,30 @@
       border: 0;
     }
 
-    &:hover,
-    &:focus {
+    // Ordering of this pseudo selector is important to ensure
```

Comment:
Not sure we should have comments about the ordering, since it's just how CSS works?